Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/ui/details/items/behavior/BehaviorInfoRow.kt

```diff
@@ -65,6 +65,7 @@ class BehaviorInfoRow @JvmOverloads constructor(
     }
 
     fun setBackgroundTint(@ColorInt color: Int) {
+        @Suppress("DEPRECATION")
```

Comment:
This is because of `setColorFilter()`, right?
Maybe put it directly before: 

```kotlin
        when {
            BuildVersionWrap.hasAPILevel(Build.VERSION_CODES.LOLLIPOP_MR1) ->
                ViewCompat.setBackgroundTintList(iconBackground, ColorStateList.valueOf(color))
            else ->
                @Suppress("DEPRECATION")
                iconBackground.background.setColorFilter(color, PorterDuff.Mode.SRC_OVER)
        }
    }
```

Reply:
👍  will consider it in the next cleaning PR